build-tools/tasks/test-utils.js

```diff
@@ -18,12 +19,59 @@ const configs = {
   dom: {
     defaultExport: `export default function wrapper(root: Element = document.body) { if (document && document.body && !document.body.contains(root)) { console.warn('[AwsUi] [test-utils] provided element is not part of the document body, interactions may work incorrectly')}; return new ElementWrapper(root); }`,
     buildFinderInterface: ({ componentName }) =>
-      `find${componentName}(selector?: string): ${toWrapper(componentName)} | null;`,
+      `/**
+        * Returns the wrapper of the first ${componentName} that matches the specified CSS selector.
+        * If no CSS selector specified, returns the wrapper of the first ${componentName}.
```

Comment:
There is some inconsistency here, some descriptions use "If no CSS selector specified", some use "If no CSS selector **_is_** specified". I recommend using "If no CSS selector is specified" everywhere for a more natural reading flow. This also aligns it with the existing "If no matching[...] **_is_** found".

```suggestion
        * If no CSS selector is specified, returns the wrapper of the first ${componentName}.
```